pyqt/pyqt.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "pyqt",
+    "config-opts": [
+        "--concatenate=1",
+        "--confirm-license",
+        "--debug",
+        "--no-designer-plugin",
+        "--no-docstrings",
+        "--no-make",
+        "--no-qml-plugin",
+        "--no-tools",
+        "--verbose",
+        "--api-dir=/app/share/qt/qsci/api/python",
+        "--qmake-setting=QMAKE_CFLAGS_RELEASE='-I/usr/include/python_pyver/'",
+        "--qmake-setting=QMAKE_CXXFLAGS_RELEASE='-I/usr/include/python_pyver/'",
+        "--target-dir=/app/lib/python_pyver/site-packages",
+        "--qt-shared"
+    ],
+    "make-args": [
+        "--directory=build"
+    ],
+    "make-install-args": [
+        "--directory=build"
+    ],
+    "sources": [
+        {
+            "type": "archive",
+            "url": "https://files.pythonhosted.org/packages/3b/27/fd81188a35f37be9b3b4c2db1654d9439d1418823916fe702ac3658c9c41/PyQt5-5.15.6.tar.gz",
+            "sha256": "80343bcab95ffba619f2ed2467fd828ffeb0a251ad7225be5fc06dcc333af452",
+            "x-checker-data": {
+                "type": "pypi",
+                "name": "PyQt5"
+            }
+        },
+        {
+            "type": "file",
+            "path": "configure"
+        },
+        {
+            "type": "file",
+            "path": "../sip/sip-build-configure.in",
+            "dest-filename": "configure.sip-build"
+        },
```

Comment:
I don't think it a good idea to manually construct a configure file, as the `args` to `configure` is uncertain.  
```
processed=$(sed 's|--prefix=/app||;s|_pyver|'${_pyver}'|g' <<< $@)
sip-build $process
```
These can break easily.  

Why not just use `build-command`, it's more controllable.

Reply:
> as the args to configure is uncertain.

I disagree here, these expect autotools buildsystem. Any change here could resonate, and possibly break many modules, as there are a lot of hand-crafted configure scripts in the wild, so it's pretty much assured that only `--prefix` will be injected by flatpak-builder.  

> Why not just use build-command, it's more controllable.

Because this way we can use the `config-opts` array, the manifest looks cleaner, and we don't need to maintain the same build commands in two places, PyQt & PyQtWebEngine.  
A script to deal with dynamically choosing the enabled PyQt bindings will still be needed, as I want to keep the option of using this repository as a shared module, so not having `configure.sip-build` won't mean that there won't be a sip-build wrapper.

Reply:
@catsout is it alright to mark this as resolved?

Reply:
Yes.

Reply:
Thanks!